doc/reference/network_physical.md

```diff
@@ -1,30 +1,58 @@
 (network-physical)=
 # Physical network
 
-The physical network type allows one to specify presets to use when connecting OVN networks to a parent interface or to allow an instance to use a physical interface as a NIC.
-This allows the instance NIC itself to simply specify the `network` it is connecting to without knowing any of the underlying configuration details.
+<!-- Include start physical intro -->
+The `physical` network type connects to an existing physical network, which can be a network card or a bridge, and serves as an uplink network for OVN.
+<!-- Include end physical intro -->
+
+This network type allows to specify presets to use when connecting OVN networks to a parent interface or to allow an instance to use a physical interface as a NIC.
+In this case, the instance NICs can simply set the `network`option to the network they connect to without knowing any of the underlying configuration details.
 
 (network-physical-options)=
 ## Configuration options
 
+The following configuration key namespaces are currently supported for the `physical` network type:
+
+ - `bgp` (BGP peer configuration)
+ - `dns` (DNS server and resolution configuration)
+ - `ipv4` (L3 IPv4 configuration)
+ - `ipv6` (L3 IPv6 configuration)
+ - `maas` (MAAS network identification)
+ - `ovn` (OVN configuration)
+ - `user` (free-form key/value for user metadata)
+
+```{note}
+{{note_ip_addresses_CIDR}}
+```
+
+The following configuration options are available for the `physical` network type:
+
 Key                             | Type      | Condition             | Default                   | Description
 :--                             | :--       | :--                   | :--                       | :--
+gvrp                            | boolean   | -                     | false                     | Register VLAN using GARP VLAN Registration Protocol
+mtu                             | integer   | -                     | -                         | The MTU of the new interface
+parent                          | string    | -                     | -                         | Parent interface to create `physical` NICs on
```

Comment:
This said "sriov NICs" before - correct to change it?

Reply:
Ah yes it was wrong before, but it should be:

> Existing interface to use for network

Rather than indicating that it is only used for `physical` NICs (it could only be used for 1 physical NIC at a time anyway) and it can be used as the uplink for OVN networks too.